examples/websocket-client-basic-auth/websocket_client_basic_auth.md

```diff
@@ -1,6 +1,6 @@
 # WebSocket client - Basic authentication
 
-A client, which is secured with Basic authentication can be used to connect to a secured service. The client is enriched with the `Authorization: Basic <token>` header by passing the `websocket:CredentialsConfig` for the `auth` configuration of the client.
+The `websocket:Client` can connect to a service that is secured with basic authentication by adding the `Authorization: Basic <token>` header to the initial HTTP request. The username and password for basic authentication can be specified in the `auth` field of the client configuration. Use this to communicate with the service, which is secured with basic authentication.
```

Comment:
Lets remove this. It is kind repetitive. 

Use this to communicate with the service, which is secured with basic authentication.